tests/x509/limbo/test_limbo.py

```diff
@@ -0,0 +1,121 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+import datetime
+import json
+import os
+from ipaddress import IPv4Address
```

Comment:
Do we have IPv6 tests in limbo as well?

Reply:
Nope, not yet. Triaging here: https://github.com/trailofbits/x509-limbo/issues/1#issuecomment-1819364991

Reply:
Added here: https://github.com/trailofbits/x509-limbo/pull/95